php_phongo.c

```diff
@@ -1051,21 +1060,18 @@ void php_phongo_read_preference_to_zval(zval *retval, mongoc_read_prefs_t *read_
 
 void php_phongo_write_concern_to_zval(zval *retval, mongoc_write_concern_t *write_concern) /* {{{ */
 {
-	char                   *wtag;
+	const char *wtag = mongoc_write_concern_get_wtag(write_concern);
 
 	array_init_size(retval, 5);
 
-	if (mongoc_write_concern_get_w(write_concern) == MONGOC_WRITE_CONCERN_W_DEFAULT) {
-		return;
```

Comment:
This was relatively intentional.
If w=MONGOC_WRITE_CONCERN_W_DEFAULT, and fsync and j aren't specifcally provided, then there is no writeConcern.
In pre-2.6 it means that getLastError will be called, with no arguments, and the server defaults applied.
In 2.6+ it means no writeConcern document provided with the write, and the server defaults applied

I'm not sure how mongoc deals with fsync and j defaults.. I guess it doesn't care if they are explicitly provided or not (e.g. overwrite server defaults if explicitly provided).......